import { Test_Invoking_Error } from '../consts/exception-message.const';

export const errorMocks: Error = new Error(Test_Invoking_Error);
